CMakeLists.txt

```diff
@@ -59,7 +59,7 @@ check_include_file (sys/types.h HAVE_SYS_TYPES_H)
 check_include_file (sys/stat.h HAVE_SYS_STAT_H)
 
 # eigen
-find_package (Eigen3 NO_MODULE)
+find_package (Eigen3 NO_MODULE QUIET)

```

Comment:
this has nothing to do with tbb support and should be in a separate PR, please describe what you're attempting to fix

Reply:
> this has nothing to do with tbb support and should be in a separate PR, please describe what you're attempting to fix

In the machines that I tried (Ubuntu 15.10 and 16.04), if I do not put `QUIET` I get fatal errors for the cmake configure process. On the contrary, putting the `QUIET` does not influence the behavior of the cmake procedure, because:

- If Eigen3 can be found with the `NO_MODULE` option, then `EIGEN3_FOUND` is set and the check in the following [line](https://github.com/resibots/libcmaes/blob/2ddb3954f735db0340b256ee5a33a060bd523f98/CMakeLists.txt#L63) is true and Eigen3 is properly included.
- If Eigen3 cannot be found with the `NO_MODULE` option or if cmake outputs a *quiet* error, then `EIGEN3_FOUND` is **NOT** set and as such the [else statement](https://github.com/resibots/libcmaes/blob/2ddb3954f735db0340b256ee5a33a060bd523f98/CMakeLists.txt#L66-L69) is executed and the Eigen3 package is searched (note that in the else statement, the Eigen package is `REQUIRED`).

Reply:
Regardless, this is another PR.